examples/incremental_learning/helmet_detection_incremental_train/README.md

```diff
@@ -24,29 +24,33 @@ matplotlib==3.3.3
 
 Follow the [Sedna installation document](/docs/setup/install.md) to install Sedna.
 
-### Prepare Data and Model
+### Prepare Model
+In this example, we need to prepare base model and deploy model in advance.
 
-* step 1: download [dataset](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/dataset.tar.gz)
-```
-mkdir -p /data/helmet_detection
-wget https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/dataset.tar.gz
-tar -zxvf dataset.tar.gz
-```
 
-* step 2: download [base model](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/model.tar.gz)
+
+download [models](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/model.tar.gz), including base model and deploy model.
 ```
 mkdir /model
```

Comment:
since there are more than one model, `/models` is better than `/model`

Reply:
make sense